network/bridge/docker/docker.go

```diff
@@ -0,0 +1,5 @@
+package docker
+
+type Docker struct {
+	Image string `json:"image" yaml:"image"`
```

Comment:
We have to inject the Docker image of https://github.com/giantswarm/k8s-network-bridge to be used in the Flannel operator. I want to do it right straight away.